cms-2013-collision-datasets-hi/inputs/hlt-trigger-path.txt

```diff
@@ -0,0 +1,702 @@
+High-Level Trigger path information HLT_Activity_Ecal_SC7
```

Comment:
The format is OK, although for better separation and readability we may want to add empty line before "High-Level Trigger path information.".

BTW I have split 2013 directory into `cms-2013-collision-datasets-hi-ppref` and `cms-2013-collision-datasets-hi` separate directories in my local WIP branch, as we had discussed about it live.

How would it affect the HLT trigger listing?  Could we easily split them, or we'll have them in one place and use symlinks?

Reply:
I'd rather have these listing in one place (with symlinks) as they come from the same source which does not distinguish the collision type.

Fine for adding the line. Can you easily do it when you're on it? I can do it is as well.

Reply:
OK, I'll merge "as is" and then add separating lines whilst creating HTL trigger path records.